File: easy/fruit/fruit.java
package ejercicios;

import java.util.Scanner;

/**
 *06/05/2021
 * @author Andre Peralta
 */
public class metodo {
    public static void main(String[] args) {
        System.out.println("Number of fruits :");
        Scanner scan = new Scanner(System.in);
        
        int fruit = scan.nextInt();
        
        int bananas = fruit/2;
        
        int cakes = (bananas-bananas%3)/3;
        
        System.out.println(cakes);

   }
}

File: easy/fruit/fruit.kt

fun cakes(fruit:Int): Int{
    //Returns the number of cakes that can be made
	
    var apples = fruit/2
    
    var number_cakes = apples/3 
    
    return(number_cakes)
}



fun main(args: Array<String>){
	    
    println("Number of fruits : ")
    var fruit = redLines()!!.toInt()
   
	println(cakes(fruit))
    

}
